src/service/history-impl.js

```diff
@@ -866,6 +866,7 @@ export function installHistoryServiceForDoc(ampdoc) {
   registerServiceBuilderForDoc(
       ampdoc,
       'history',
-      /* opt_constructor */ undefined,
-      ampdoc => createHistory(ampdoc));
+      function(ampdoc) {
```

Comment:
Can't this be just `registerServiceBuilderForDoc(..., createHistory)`? Why reclosure the function with same args?